pex/common.py

```diff
@@ -264,17 +270,8 @@ def link(self, src, dst, label=None):
     self._ensure_parent(dst)
     abs_src = src
     abs_dst = os.path.join(self.chroot, dst)
-    try:
-      os.link(abs_src, abs_dst)
-    except OSError as e:
-      if e.errno == errno.EEXIST:
-        # File already exists, skip XXX -- ensure target and dest are same?
-        pass
-      elif e.errno == errno.EXDEV:
-        # Hard link across devices, fall back on copying
-        shutil.copyfile(abs_src, abs_dst)
-      else:
-        raise
+    safe_copy(abs_src, abs_dst, overwrite=False)
+    # If the file already exists, skip XXX -- ensure target and dest are same?
```

Comment:
mind making this a TODO and rewording it a bit to make sense in the modified context?